torch/_inductor/cpp_builder.py

```diff
@@ -1300,6 +1301,27 @@ def _load_icx_built_in_lib_by_name(cpp_compiler: str, lib_name: str) -> bool:
         _load_icx_built_in_lib_by_name(cpp_compiler, lib_name)
 
 
+def preload_gcc_libgomp_linux(cflags, ldflags, libs, lib_dir_paths):
```

Comment:
It seems that `cflags` and `lib_dir_paths` are not being used. Can we remove them if they are not needed?